lib/openscap_parser/profile.rb

```diff
@@ -52,12 +52,33 @@ def refined_values
       end
     end
 
+    def refined_rules(attribute)
+      @refined_rules ||= @parsed_xml.xpath("refine-rule[@#{attribute}]").each_with_object({}) do |element, rr|
+        rr[element.at_xpath('@idref').text] = element.at_xpath("@#{attribute}")&.text
+      end
+    end
+
+    def refined_rule_severity
+      refined_rules('severity')
+    end
+
+    def refined_rule_role
+      refined_rules('role')
+    end
+
+    def refined_rule_weight
+      refined_rules('weight')
+    end
```

Comment:
@skateman Does this look good to you for parsing refined rules? These are the only 3 attributes of a rule that can be refined

Reply:
Do we use the role/weight params elsewhere? If no then we might not want to refine these...

Reply:
`role` is only a Rule attribute but `weight` can be used for a Rule or Group. Should we only deal with severity here too though since we don't actually use `role` or `weight` or should we parse them anyway?